src/Form/QuestionType/QuestionTypeInterface.php

```diff
@@ -63,6 +63,16 @@ public static function formatDefaultValueForDB(mixed $value): ?string;
      */
     public static function validateExtraDataInput(array $input): bool;
 
+    /**
+     * Prepare the answer for the end user.
+     * This method is called before saving the answer.
+     *
+     * @param Question $question The question data.
+     * @param mixed $answer The raw answer data.
+     * @return mixed The prepared answer data.
+     */
+    public static function prepareEndUserAnswer(Question $question, mixed $answer): mixed;
```

Comment:
Please avoid `static` methods. Even if there is no class properties for now, we may add some in the future, and we would need a non-static method to be able to access them. It is preferable to use a non static method right now, even if it is not yet usefull, as changing it later would be a major BC break.

By the way, other methods from the interface should probably also be changed from static to non static.

```suggestion
    public function prepareEndUserAnswer(Question $question, mixed $answer): mixed;
```